circle.yml

```diff
@@ -5,3 +8,13 @@ dependencies:
     - "target/streams"
     - "project/target/resolution-cache"
     - "project/target/streams"
+  override:
+    - sbt test:compile
+test:
+  override:
+    - sbt test
+  post:
+    - sbt riffRaffUpload
```

Comment:
The project doesn't yet have tests so I wouldn't have noticed.
I will go back and apply this change to the other CircleCI projects I have done as a job lot.